ethereum/.env.arbitrum.sepolia.testnet

```diff
@@ -0,0 +1,17 @@
+# Arbitrum on Sepolia testnet read only env
+# Rename to .env to use with truffle migrations
+
+# Wormhole Core Migrations
+INIT_SIGNERS=["0x13947Bd48b18E53fdAeEe77F3473391aC727C638"]
+INIT_CHAIN_ID=10003
+INIT_GOV_CHAIN_ID=0x1
+INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+INIT_EVM_CHAIN_ID=421614
+
+# Bridge Migrations
+BRIDGE_INIT_CHAIN_ID=10003
+BRIDGE_INIT_GOV_CHAIN_ID=0x1
+BRIDGE_INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+# Not sure if there should be a WETH.
+BRIDGE_INIT_WETH=0x0000000000000000000000000000000000000000
```

Comment:
The L2 WETH is `0x980B62Da83eFf3D4576C647993b0c1D7faf17c73`
https://docs.arbitrum.io/for-devs/useful-addresses#token-bridge-smart-contracts

Reply:
Updated the contract to reflect this.